internal/resources/app_endpoint_oidc_schema.go

```diff
@@ -0,0 +1,87 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+)
+
+func AppEndpointOidcProviderSchema() schema.Schema {
+	return schema.Schema{
+		MarkdownDescription: "App Endpoint OpenID Connect Provider Resource",
+		Attributes: map[string]schema.Attribute{
+			"app_endpoint_name": schema.StringAttribute{
+				Required: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplaceIfConfigured(),
+				},
+				MarkdownDescription: `The name of the App Endpoint. Requires replacement if changed.`,
+			},
+			"app_service_id": schema.StringAttribute{
+				Required: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplaceIfConfigured(),
```

Comment:
`RequiresReplaceIfConfigured` is not needed since you're already checking if this value is null

Reply:
https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/396/commits/4b3c7e572db6d00a3b3a59f7edd2557c541930d6